Earthfile

```diff
@@ -171,13 +170,16 @@ rosdep:
       --mount=type=cache,mode=0777,target=/var/lib/apt,sharing=locked,id=lib_apt_cache \
       sudo apt-get update && \
       rosdep update && \
-      rosdep install -y \
+      rosdep install -s -y \
         --from-paths src --ignore-src \
         --rosdistro ${ROS_DISTRO} \
         # `urdfdom_headers` is cloned from source, however rosdep can't find it.
         # It is because package.xml manifest is missing. See: https://github.com/ros/urdfdom_headers
         # Additionally, IKOS must be excluded as per: https://github.com/space-ros/docker/issues/99
-        --skip-keys "$(tr '\n' ' ' < 'excluded-pkgs.txt') urdfdom_headers ikos"
+        --skip-keys "$(tr '\n' ' ' < 'excluded-pkgs.txt') urdfdom_headers ikos" > rosdep-commands.sh && \
+      chmod u+x rosdep-commands.sh && \
+      ./rosdep-commands.sh
```

Comment:
If we want to cache rosdep better (in a later PR), we could end this at the `> rosdep-commands.sh` and then `COPY` that file into another layer and install there - then the install wouldn't be re-run if the script didn't change.